### PR TITLE
fix accessibility footer links & images

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: src/Template/Element/Layout/default/footer/social_links.ctp [hunk 1]
```diff
@@ -75,6 +75,7 @@
 		'target' => '_blank',
 		'title' => 'IRC',
 		'class' => 'icon-irc',
-		'data-toggle' => 'tooltip'
+		'data-toggle' => 'tooltip',
+        'aria-label' => 'Stack Overflow'
```
Comment (garas) [inline src/Template/Element/Layout/default/footer/social_links.ctp:79]: It is link to IRC

File: src/Template/Element/Layout/default/footer/login_form.ctp [hunk 2]
```diff
@@ -26,13 +26,13 @@
             <?= $this->Html->link(
     $this->Html->image('open-hub.svg'),
     'https://www.openhub.net/p/cakephp',
-    ['escape' => false, 'style' => 'opacity: .5']
+    ['escape' => false, 'style' => 'opacity: .5', 'aria-label' => 'openHub']
```
Comment (garas) [inline src/Template/Element/Layout/default/footer/login_form.ctp:29]: `` would be better here.

File: src/Template/Element/Layout/default/footer/social_links.ctp (hunk 1 above)
Comment (Arhell) [reply src/Template/Element/Layout/default/footer/social_links.ctp:79]: Thanks, I will correct

File: src/Template/Element/Layout/default/footer/login_form.ctp (hunk 2 above)
Comment (Arhell) [reply src/Template/Element/Layout/default/footer/login_form.ctp:29]: I will also add

File: src/Template/Element/Layout/default/footer/login_form.ctp [hunk 3]
```diff
@@ -26,13 +26,13 @@
             <?= $this->Html->link(
     $this->Html->image('open-hub.svg'),
     'https://www.openhub.net/p/cakephp',
-    ['escape' => false, 'style' => 'opacity: .5']
+    ['escape' => false, 'style' => 'opacity: .5', 'aria-label' => 'openHub', 'alt' => 'openHub']
```
Comment (garas) [inline src/Template/Element/Layout/default/footer/login_form.ctp:29]: `'alt' => 'openHub'` should go to `$this->Html->image()` above.

**Pull request merged**